Tella/Data/Database/Vault/VaultDataBase.swift

```diff
@@ -146,7 +163,39 @@ class VaultDatabase : DataBase, VaultDataBaseProtocol {
             return nil
         }
     }
+
+    func getNonUpdatedVaultFiles() -> [VaultFileDB] {
+        do {
+            let vaultFilesDict = try statementBuilder.getSelectQuery(tableName: VaultD.tVaultFile,
+                                                                     equalCondition:[KeyValue(key: VaultD.cEncryptionUpdated, value: false)])
+            
+            let vaultFiles = vaultFilesDict.compactMap{VaultFileDB.init(dictionnary: $0)}
+            return vaultFiles
+            
+        } catch {
+            return []
+        }
+    }
     
+    @discardableResult
+    func updateEncryptionVaultFile(id: String?) -> Result<Bool, Error> {
```

Comment:
`id` is nullable -> let use guard to ensure that the id is not nil, throwing a meaningful error if it is. what do you think?